src/components/TableUser/data-table.tsx

```diff
@@ -70,7 +70,7 @@ export function DataTable<TData, TValue>({
   });
 
   return (
-    <div className="grid gap-2">
+    <div className="grid gap-2 grid-cols-1">
```

Comment:
Do we need to explicitly adding class `w-full overflow-x-auto`?

Reply:
`overflow-x-auto` is unnecessary since we already wrap the table component with `overflow-auto` for scrollable behaviour.

`w-full` is not needed either because naturally this div will fit its container. 

Cmiiw, mas. 🙏